theory.tex

```diff
@@ -93,19 +137,20 @@ \subsection{Centrifuges basic properties and definition}
 \subsection{Cascade Design}
 \subsubsection{Symmetric Cascade}
 
-A symmetric cascade is a cascade where a stage is feed, $F$, with the tail, $T$, of the next
-stage, the product, $P$, of the previous one, and for the cascade feeding stage
-an external feed, $F_{ext}$, i.e. :
+A symmetric cascade is a cascade where a stage's feed, $F$, with a tail, $T$, of the next
+stage and product, $P$, of the previous one. For the cascade feeding stage
+and external feed, $F_{ext}$:
 
 \begin{equation}
     F_{i} = T_{i+1} + F_{i-1} ~(+ F_{ext})
```

Comment:
Is `F_{i-1}` meant to be `P_{i-1}`, or am I misreading?
```suggestion
    F_{i} = T_{i+1} + P{i-1} ~(+ F_{ext})
```

Reply:
this sounds about right

Reply:
but you missed the `_`

Reply:
Good question... @bam241 might want to confirm this but I think you're right. The feed of stage `i` should be made up of the product of the upper stage plus the feed of the lower stage. If this was `F_{i-1}` it would ignore the output of that stage (?)

Reply:
You beat me to it! So,
```suggestion
    F_{i} = T_{i+1} + P_{i-1} ~(+ F_{ext})
```